package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -68,7 +68,7 @@ def search(self, atoms, radius, level='A'):
 
         Parameters
         ----------
-        atoms : AtomGroup or Atom
+        atoms : AtomGroup or `Atom`
```

Comment:
Would `Atom` be recognized if the types were separated by comma and not "or", i.e.,
```reST
atoms : AtomGroup, Atom
```